app/policies/person_policy.rb

```diff
@@ -3,42 +3,38 @@ class Scope < Scope
     def resolve
       case
       when sys_admin? || admin?
-        original_scope.all
-      when acting_user.present?
-        original_scope.where(user_id: acting_user.id)
+        scope.all
+      when user.present?
+        scope.where(user_id: user.id)
       else
-        original_scope.none
+        scope.none
       end
     end
   end
 
   def read?
-    person_attached_to_acting_user? ||
-      sys_admin? ||
-      admin?
+    own_person? || sys_admin? || admin?
```

Comment:
`own_person` takes me a minute to translate, but, I don't have a different suggestion atm that doesn't have a namespace issue. :)